lib/rihanna/job.ex

```diff
@@ -96,11 +105,11 @@ defmodule Rihanna.Job do
     result =
       producer_query(
         """
-          INSERT INTO "#{table()}" (term, enqueued_at, due_at)
-          VALUES ($1, $2, $3)
+          INSERT INTO "#{table()}" (term, enqueued_at, due_at, priority)
+          VALUES ($1, $2, $3, $4)
           RETURNING #{@sql_fields}
         """,
-        [serialized_term, now, due_at]
+        [serialized_term, now, due_at, priority()]
```

Comment:
How is the user expected to provide this value? It seems to be hard coded to 19.

Reply:
They can override it by implementing a module with the `Rihanna.Job` behaviour and overriding this value:

```elixir
def priority, do: 5
```

Reply:
The priority function is being called on the behaviour module not the implementation module so it cannot be overridden in this way. It would need to be added to the behaviour as an optional callback, and then it would be called on implementation module if present.

A test for this would be good :)

Reply:
This still needs attention :)

Reply:
Yeah … this needs to be `@callback`, or maybe set when enqueueing a job. I prefer the latter, but it could be both, with it defaulting to the Job module or some default.